extensions/sql-migration/src/wizard/skuRecommendationPage.ts

```diff
@@ -557,6 +580,11 @@ export class SKURecommendationPage extends MigrationWizardPage {
 					e => `message: ${e.message}${EOL}errorSummary: ${e.errorSummary}${EOL}possibleCauses: ${e.possibleCauses}${EOL}guidance: ${e.guidance}${EOL}errorId: ${e.errorId}`)!);
 			}
 
+			const skuRecommendationError = this.migrationStateModel._skuRecommendationResults?.recommendationError;
+			if (skuRecommendationError) {
+				errors.push(`message: ${skuRecommendationError.message}${EOL}stack: ${skuRecommendationError.stack}`);
```

Comment:
what's an example of a sku recommendation error? This error list will be surfaced with the user as a banner, like this:
![image](https://user-images.githubusercontent.com/14362577/150039552-a06053e6-c025-49b0-b68b-7122e33f1233.png)

I want to make sure the errors are user friendly errors

Reply:
Currently errors are getting surfaced properly, which is the key thing, but not in a user friendly way yet (because I don't know exactly what guidance, wording, etc. we want to use). Pretty much the only main "error" you can run into with SKU recommendation is when the perf data is missing, and currently that looks like this:

![image](https://user-images.githubusercontent.com/11844501/150041777-668fdbe8-b77a-4439-8708-c7cfd43189a7.png)

So the correct errors are being surfaced, now just a matter of maybe catching them and re-throwing them with a better message